apps/adresse-service/src/main/java/no/nav/testnav/apps/adresseservice/AdresseServiceApplicationStarter.java

```diff
@@ -1,13 +1,16 @@
 package no.nav.testnav.apps.adresseservice;
 
-import org.springframework.boot.SpringApplication;
+import no.nav.testnav.libs.servletcore.config.NaisEnvironmentApplicationContextInitializer;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
 
 @SpringBootApplication
 public class AdresseServiceApplicationStarter {
 
     public static void main(String[] args) {
-
-        SpringApplication.run(AdresseServiceApplicationStarter.class, args);
+        new SpringApplicationBuilder(AdresseServiceApplicationStarter.class)
+                .initializers(new NaisEnvironmentApplicationContextInitializer())
```

Comment:
Jeg ville nok heller hatt denne i ApplicationConfig, den finnes allerede og latt ApplicationStarter være så enkel som mulig.

Reply:
Application context initializer må kalles før Spring context er ferdig opprettet, så jeg ser ikke hvordan det skal være teknisk mulig. 🤔 Bruker de eksisterende APIene i Spring for å gjøre dette (de nye APIene, siden `DelegatingApplicationContextInitializer` er fjernet f.o.m. Spring Boot 3.4.0) Vi setter config gjennom kode avhengig av hvilken Spring profile som er satt for initialisering.

Reply:
Skjønner, vel det var et forslag, hvis det ikke er mulig, så kjør på. 😄